pkgs/by-name/pv/pv-migrate/package.nix

```diff
@@ -3,6 +3,7 @@
   buildGoModule,
   fetchFromGitHub,
   installShellFiles,
+  gitUpdater,
```

Comment:
this isn't needed -- nix-update will work fine for updating github releases
https://nix-community.github.io/nixpkgs-update/nixpkgs-maintainer-faq/#nixpkgs-maintainer-faq

Reply:
ok

Reply:
btw is there a central way to check what packages do not have auto updater capability yet?

Reply:
I don't know. I usually look at the logs https://nixpkgs-update-logs.nix-community.org/ eg: https://nixpkgs-update-logs.nix-community.org/pv-migrate but It looks like it only logs when there is an update. if there is an update script I think it will log every time the script is run. but I don't think that helps find what you're looking for.